test/inductor/test_torchinductor.py

```diff
@@ -3764,6 +3764,35 @@ def fn2(a):
             (torch.randn([2, 2, 10]),),
         )
 
+    def test_low_memory_max_pool(self):
+        prims = torch.ops.prims
+
+        def fn(x):
+            kernel_size = [3, 3]
+            stride = [2, 2]
+            padding = [1, 1]
+            dilation = [1, 1]
+            ceil_mode = False
+
+            vals, offsets = prims._low_memory_max_pool2d_with_offsets(
+                x,
+                kernel_size,
+                stride,
+                padding,
+                dilation,
+                ceil_mode,
+            )
+            indices = prims._low_memory_max_pool2d_offsets_to_indices(
+                offsets,
+                kernel_size[1],
+                x.size(-1),
+                stride,
+                padding,
+            )
+            return vals, indices, offsets
+
+        self.common(fn, (torch.randn(1, 3, 10, 10),))
```

Comment:
Won't you need to use the aot decomp partition backend to hit the new code?

Reply:
No, this test checks the eager implementation vs the lowering. The new code is used in the eager implementation.